manager/volume.go

```diff
@@ -613,6 +616,46 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 		return nil, fmt.Errorf("cannot expand volume before replica scheduling success")
 	}
 
+	kubernetesStatus := &v.Status.KubernetesStatus
+	if kubernetesStatus.PVCName != "" && kubernetesStatus.LastPVCRefAt == "" {
+		pvc, err := m.ds.GetPersistentVolumeClaim(kubernetesStatus.Namespace, kubernetesStatus.PVCName)
+		if err != nil {
+			return nil, err
+		}
+
+		resourceSize := resource.MustParse(strconv.FormatInt(size, 10))
+
+		// TODO: Should check for pvc.Spec.Resources.Requests.Storage() here, once upgrade API to v0.18.x.
+		storageSpecValue := pvc.Spec.Resources.Requests[corev1.ResourceStorage]
```

Comment:
Please do map index check to avoid null pointer error.

Reply:
changed